docs/tutorials/semantic_search.md

```diff
@@ -0,0 +1,207 @@
+# Semantic Search
+
+## Context
+
+The Pixano's app support **semantic search** which allows a user to search for a view based on a text content.
+
+To do that Pixano requires that the embeddings of the views are pre-computed with the model used for Semantic search. For now, we rely on the [Embeddings functions](https://lancedb.github.io/lancedb/embeddings/embedding_functions/) from LanceDB.
+
+## Open-clip example
+
+We will go through the process of semantic search based on the Open-clip model.
+
+### Pre-compute the embeddings
+
+First, we need to pre-compute the embeddings using LanceDB and Pixano. We will use the Health Images dataset defined in the [library](./create_your_first_library.md) tutorial.
+
+0. Install open-clip
+
+For this tutorial, we will use open-clip embedding function and therefore need it to be installed.
+
+```bash
+pip install open_clip_torch
+```
```

Comment:
simple remark: you don't prefer to show the hugging face way ?

Reply:
The embedding function are done by LanceDB it is not my choice.